l10n_it_fatturapa/models/partner.py

```diff
@@ -70,30 +70,44 @@ def _check_ftpa_partner_data(self):
                         "As a natural person, partner %s "
                         "must have Name and Surname"
                     ) % partner.name)
-                if not partner.is_pa and (
-                    not partner.codice_destinatario or
-                        len(partner.codice_destinatario) != 7):
-                    raise ValidationError(_("Partner %s Addressee Code"
-                                            " must be 7 characters long")
-                                          % partner.name)
-                if not partner.vat and not partner.fiscalcode:
+                if not partner.country_id:
```

Comment:
Questo controllo è già presente: https://github.com/OCA/l10n-italy/blob/8.0/l10n_it_fatturapa/models/partner.py#L95

Reply:
si, è stato spostato da L95 a L73 perchè va controllato prima

Reply:
ok, sarei però per introdurre questo cambiamento con un'altra PR, in modo che il commit sia facilmente applicabile anche sulle altre versioni.

Reply:
mm, mi pareva fosse già stato fatto sulle altre

Reply:
Mi pare ancora in fondo.

* v10.0: https://github.com/OCA/l10n-italy/blob/10.0/l10n_it_fatturapa/models/partner.py#L106
* v11.0: https://github.com/OCA/l10n-italy/blob/11.0/l10n_it_fatturapa/models/partner.py#L105
* v12.0: https://github.com/OCA/l10n-italy/blob/12.0/l10n_it_fatturapa/models/partner.py#L105

Reply:
@sergiocorato rimarrebbe solo più questo punto. Riesci a darmi un feedback?
Grazie in anticipo.

Reply:
@sergiocorato scusa se ti sollecito ma avrei esigenza che venga mergiata! Potresti darmi un feedback?

Reply:
eccomi, tolgo la modifica e la metto su altra PR

Reply:
fatto